### ISSUE TITLE
Footer still missing

### ISSUE THREAD

**Issue by almereyda** [TransforMap/transformap.github.io#3]
Maybe simple as on https://mozilla.org ?

**Comment by gandhiano** [comment]
Yes, something like that. Maybe with the partners logos on a row. And taking out links to not clearly related stuff like this one: http://directory.open-steps.org/

**Comment by almereyda** [comment]
Until we run our own PLP directory, we could point the link to http://discourse.transformap.co/groups/Team/members

But I'd ask you to wait for my return to Berlin before any big changes.
